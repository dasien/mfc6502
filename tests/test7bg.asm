; Testing increment, decrement, and compare.
LDX #01
INX
INX
LDY #01
INY
INY
DEX
DEY
DEY