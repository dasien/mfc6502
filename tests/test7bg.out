A2 01
E8 
E8 
A0 01
C8 
C8 
CA 
88 
88 
